espei/utils.py

```diff
@@ -45,7 +45,8 @@ def __getstate__(self):
 
     def __setstate__(self, state):
         self.__init__(storage=MemoryStorage)
-        self.insert_multiple(state['_table'])
+        print(state)
```

Comment:
print statement slipped in here

Reply:
https://github.com/PhasesResearchLab/ESPEI/commit/50d92e877a1b92e9afd4eab72f313d352e5d6326 grabbed it, thanks!